app/jobs/bulkrax/delete_job.rb

```diff
@@ -5,8 +5,10 @@ class DeleteJob < ApplicationJob
     queue_as :import
 
     def perform(entry, importer_run)
-      obj = entry.factory.find
-      obj&.delete
+      obj = entry.factory.class.find(entry.raw_metadata["id"])
```

Comment:
I'm confused, why can we do `entry.factory.find`?

It seams like we could be able to call `entry.factory.delete(user: importer_run.importer.user)` and it would do the right thing.